salt/grains/core.py

```diff
@@ -1957,7 +1957,9 @@ def os_data():
             # /etc/os-release contains no minor distro release number so we fall back to parse
             # /etc/centos-release file instead.
             # Commit introducing this comment should be reverted after the upstream bug is released.
-            if 'CentOS Linux 7' in grains.get('lsb_distrib_codename', ''):
+            # This continues with CentOS 8, same bug is still assigned.
+            if ('CentOS Linux 7' in grains.get('lsb_distrib_codename', '')
+                or 'CentOS Linux 8' in grains.get('lsb_distrib_codename', '')):
```

Comment:
2 major releases, perhaps this should be considered feature and this check should assume the following will remind true going forward. Therefore, generalize it.

Reply:
I'm inclined to agree based on the response (and lack thereof) in the CentOS bug. Are you suggesting Salt checks for 'CentOS Linux 6' (or 6 or 5) and this becomes `if not`? Or something else?

Reply:
Essentially, `>= 7`. I assume everything is fine below 7.

Reply:
Mentioned in PR, but this behavior is tested the same back to and including CentOS 6